src/pages/docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -85,19 +87,19 @@ Once the variables for target discovery have been configured the **Amazon Web Se
 
 Octopus looks for tags applied to cloud resources to discover and create deployment targets for you.
 
-Tags are in the format `octopus-{scope}` and support the following for discovery. Note that only a single value is supported in tags at the moment.
+Tags in cloud resource templates are in the format `octopus-{scope}` and support the following for discovery. Note that only a single value is supported in tags at the moment.
 
 | Tag                   | Required | Description                                                                                                                                                                                  | Example                             |
 | --------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
 | `octopus-environment` | Yes      | The name of the [environment](/docs/infrastructure/environments) the target can be used in during deployments. Only deployments matching the environment will discover the target.  | `octopus-environment = Development` |
-| `octopus-role`        | Yes      | The [role](/docs/infrastructure/deployment-targets/#target-roles) that should be applied to the target. Only deployments with a step that matches the role will discover the target. | `octopus-role = PetShotFrontEnd`    |
+| `octopus-role`        | Yes      | The [target tag](/docs/infrastructure/deployment-targets/#target-roles) that should be applied to the target. Only deployments with a step that matches the target tag will discover the target. | `octopus-role = PetShotFrontEnd`    |
```

Comment:
Let's capture that this is still called `octopus-role` as something to potentially update in the future.

Reply:
I've captured it in Shortcut here: https://app.shortcut.com/octopusdeploy/story/77510/consider-potentially-renaming-target-roles-to-target-tags-in-templates-used-for-cloud-target-discovery-and-cli-tools